docs/overview/deep_dive/function_types.rst

```diff
@@ -181,7 +181,7 @@ This is another categorization which is **not** mutually exclusive, as outlined
 
 Primary convenience functions include: `ivy.can_cast`_ which determines if one data type can be cast to another data type according to type-promotion rules, `ivy.dtype`_ which gets the data type for the input array, and `ivy.dev`_ which gets the device for the input array.
 
-Compositional convenience functions include: `ivy.set_default_dtype`_ which sets the global default data dtype, `ivy.default_dtype`_ which returns the correct data type to use, considering both the inputs and also the globally set default, and `ivy.get_all_arrays_on_dev`_ which gets all arrays which are currently on the specified device.
+Compositional convenience functions include: `ivy.set_default_dtype`_ which sets the global default data dtype, `ivy.default_dtype`_ which returns the correct data type to use, considering both the inputs and the default data type set globally, and `ivy.get_all_arrays_on_dev`_ which gets all arrays which are currently on the specified device.
```

Comment:
I'm not sure about your modification. I would suggest "and the globally set default data type" or "and the global default data type". What do you think?

Reply:
thanks for the review, 
I think "and the globally set default data type" is better.